addons/account/models/account_move.py

```diff
@@ -2184,11 +2189,11 @@ def _get_move_display_name(self, show_ref=False):
                 'in_receipt': _('Draft Purchase Receipt'),
                 'entry': _('Draft Entry'),
             }[self.move_type]
-            if not self.name or self.name == '/':
-                draft_name += ' (* %s)' % str(self.id)
-            else:
-                draft_name += ' ' + self.name
-        return (draft_name or self.name) + (show_ref and self.ref and ' (%s%s)' % (self.ref[:50], '...' if len(self.ref) > 50 else '') or '')
+        if not self.name or self.name == '/':
+            name += ' (* %s)' % str(self.id)
+        else:
+            name += ' ' + self.name
+        return name + (show_ref and self.ref and ' (%s%s)' % (self.ref[:50], '...' if len(self.ref) > 50 else '') or '')
```

Comment:
Now the display name always starts with a space when not in draft

Reply:
moved the space